schemas/src/definitions/extensionType.yaml

```diff
@@ -0,0 +1,40 @@
+# yaml-language-server: $schema=https://json-schema.org/draft/2020-12/schema
+$schema: https://json-schema.org/draft/2020-12/schema
+$id: <HOST>/<PREFIX>/<VERSION>/definitions/extensionType.yaml
+
+title: DSC extension fully qualified type name
+description: |
+  The namespaced name of the DSC extension, using the syntax:
+
+  owner[.group][.area]/name
+
+  For example:
+
+    - Microsoft.DSC/Docs
+    - Microsoft.Windows.Appx/Discover
+
+type: string
+pattern: ^\w+(\.\w+){0,2}\/\w+$
```

Comment:
This was brought up in an issue, do we really want to enforce only two dots?

Reply:
That was the original spec iirc, but we can loosen it, either leaving it open or limiting it to `n` levels of hierarchy.

Reply:
Maybe change it to allow up to 4?